woocommerce/woocommerce-paypal-payments
# ACDC checkout error: "Card Details not valid"; but payment completes

### Description
Changing billing address information in the checkout causes an error message upon order submission with the PayPal Card Processing enabled, but the order goes through: `Unfortunately, your credit card details are not valid`

### Steps to test
1. create an order with a registered user, so that checkout info is saved.
2. add new item to your cart and visit the checkout
3. modify some checkout billing fields like postal code or city name
4. enter credit card details
5. submit order
6. → error message appears up top, but the payment completes fine
7. buyer is redirected to the order-received page.

### Expected behaviour
The error message `Unfortunately, your credit card details are not valid` doesn’t make much sense when the order still completes fine.
